src/styles/menu.scss

```diff
@@ -1,4 +1,46 @@
 .menu {
+  align-items: center;
   display: flex;
   flex-direction: column;
+  width: 30%;
+  padding: 1rem;
+
+  .profile-pic {
+    position: relative;
+    height: 100px;
+    width: 100px;
+
+    img {
+      left: 50%;
+      position: absolute;
+      top: 50%;
+      transform: translate(-50%, -50%);
+    }
+  }
+
+  .edit-logout {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-size: .8rem;
+    color: $_BLACK;
```

Comment:
I saw @felipegarcia92 asked you to change sass constants to $black, so could you change the ones in this file to keep the same structure?